### PR TITLE
As a resident, when I go to the "Inji tour guide" section, I should be taken to the "Intro Slides"

### PR DESCRIPTION
**Pull request by Anil-kumar-Majji**
[INJI-25](https://mosip.atlassian.net/browse/INJI-25?atlOrigin=eyJpIjoiZmFlYTM2MjI2Y2YzNGJmZWEyYzcyNGUwMDNiNmM5MzAiLCJwIjoiaiJ9) **--->**  As a resident, when I go to the "Inji tour guide" section, I should be taken to the "Intro Slides"

### CODE REVIEW

File: screens/WelcomeScreenController.ts [hunk 1]
```diff
@@ -28,13 +31,18 @@ export function useWelcomeScreen(props: RootRouteProps) {
...
     NEXT: () => {
       authService.send(AuthEvents.NEXT()), props.navigation.navigate('Auth');
     },
     SELECT: () => {
       authService.send(AuthEvents.SELECT()),
         props.navigation.navigate('IntroSliders');
     },
+    BACK: () => {
+      settingsService.send(SettingsEvents.BACK()),
+        props.navigation.navigate('Main');
```
Comment (vharsh) [inline screens/WelcomeScreenController.ts:44]: format the code once from vscode

File: screens/Settings/SettingScreenController.ts [hunk 2]
```diff
@@ -143,6 +143,12 @@ export function useSettingsScreen(props: RequestRouteProps) {
       setIsVisible(false);
     },
 
+    INJI_TOUR_GUIDE: () => {
+      settingsService.send(SettingsEvents.INJI_TOUR_GUIDE()),
```
Comment (vharsh) [inline screens/Settings/SettingScreenController.ts:147]:
```suggestion
      settingsService.send(SettingsEvents.INJI_TOUR_GUIDE());
```

File: screens/Settings/SettingScreen.tsx [hunk 3]
```diff
@@ -154,13 +154,11 @@ export const SettingScreen: React.FC<SettingProps & RequestRouteProps> = (
                 progress={controller.isResetInjiProps}
               />
             )}
-            {/*
+
             <ListItem
```
Comment (vharsh) [inline screens/Settings/SettingScreen.tsx:158]: Please attach a screenshot for this change.

File: machines/settings.ts [hunk 4]
```diff
@@ -126,6 +129,20 @@ export const settingsMachine = model.createMachine(
...
+          },
+        },
+      },
+      showInjiTourGuide: {
+        on: {
+          BACK: {
+            target: 'idle',
+          },
+        },
+      },
```
Comment (vharsh) [inline machines/settings.ts:145]: @PuBHARGAVI Can you review the changes in the state-machine's state change for showing the tour-guide from settings?

**Review by vharsh — COMMENTED** [review]
looks good to me, I'd like @PuBHARGAVI to take a look on the changes too.

File: machines/settings.ts [hunk 5]
```diff
@@ -126,6 +129,20 @@ export const settingsMachine = model.createMachine(
           },
         },
       },
+      injiToutGuide: {
```
Comment (PuBHARGAVI) [inline machines/settings.ts:132]: @Anil-kumar-Majji state should be "injiTourGuide" not "injiToutGuide". Please make this change and push the code

Comment (Anil-kumar-Majji) [reply machines/settings.ts:132]: Done

File: screens/Settings/SettingScreen.tsx (hunk 3 above)
Comment (Anil-kumar-Majji) [reply screens/Settings/SettingScreen.tsx:158]: Attached in the comments of resp. Jira ticket

File: screens/Settings/SettingScreenController.ts (hunk 2 above)
Comment (Anil-kumar-Majji) [reply screens/Settings/SettingScreenController.ts:147]: Done

File: screens/Home/IntroSlidersScreen.tsx [hunk 6]
```diff
@@ -122,7 +130,7 @@ export const IntroSlidersScreen: React.FC<RootRouteProps> = (props) => {
             align="center"
             color="#FFFFFF"
             margin="15 0 0 0">
-            {t('stepFourButton')}
+            {controller.passcode === '' ? t('getStarted') : t('goBack')}
```
[the remark this answers is not in the document]
Comment (Anil-kumar-Majji) [reply screens/Home/IntroSlidersScreen.tsx:133]: Updated with the **isPasswcodeSet** function. Thanks for suggesting.

File: locales/hin.json [hunk 7]
```diff
@@ -264,7 +264,9 @@
     "stepThreeText": "परेशानी मुक्त तरीके से अपने कार्ड सुरक्षित रूप से साझा करें और विभिन्न सेवाओं का लाभ उठाएं।",
     "stepFourTitle": "परेशानी मुक्त प्रमाणीकरण",
     "stepFourText": "संग्रहीत डिजिटल क्रेडेंशियल का उपयोग करके आसानी से स्वयं को प्रमाणित करें।",
-    "stepFourButton": "शुरू हो जाओ",
+    "getStarted": "शुरू हो जाओ",
```
[the remark this answers is not in the document]
Comment (swatigoel) [reply locales/hin.json:267]: @vharsh right
@Anil-kumar-Majji let's add this `शुरू करें`

[the remark this answers is not in the document]
Comment (Anil-kumar-Majji) [reply locales/hin.json]: Text changed from `शुरू हो जाओ`  to ` शुरू करें`

**Review by swatigoel — APPROVED** [review]

**Review by vijay151096 — APPROVED** [review]
all the conversations are resolved, hence merging it.

**Pull request merged**